src/include/migraphx/op/pooling.hpp

```diff
@@ -331,7 +331,7 @@ struct pooling
                 int end;
                 std::size_t dilated_kernel_dim = dilate_dim(kernel_dims[d_2], dilations[d_2]);
                 // NOLINT
-                if(count_include_pad and (mode != pooling_mode::max))
+                if(count_include_pad and mode == pooling_mode::average)
```

Comment:
This would disable count_include_pad for the lp_norm

Reply:
https://onnx.ai/onnx/operators/onnx__LpNormalization.html